suite-native/transactions/src/components/TransactionsList/TokenTransferListItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { memo } from 'react';
+
+import { EthereumTokenSymbol } from '@suite-native/ethereum-tokens';
+import { AccountKey } from '@suite-common/wallet-types';
+import { EthereumTokenAmountFormatter, TokenToFiatAmountFormatter } from '@suite-native/formatters';
+import { TokenTransfer } from '@trezor/blockchain-link-types';
+
+import { TransactionListItemContainer } from './TransactionListItemContainer';
+
+type TokenTransferListItemProps = {
+    txid: string;
+    tokenTransfer: TokenTransfer;
+    accountKey: AccountKey;
+    isFirst?: boolean;
+    isLast?: boolean;
+};
+
+export const TokenTransferListItem = memo(
+    ({ txid, accountKey, tokenTransfer, isFirst, isLast }: TokenTransferListItemProps) => {
+        const tokenSymbol = tokenTransfer.symbol.toLowerCase() as EthereumTokenSymbol;
```

Comment:
This should not happen anywhere for branded types expect place where they enter codebase, otherwise they are kind of useless. You should probalby retype selector that is selecting Transactions to return this type in first place.

Reply:
selector modified in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25